helm/alfresco-process-infrastructure/values.yaml

```diff
@@ -470,7 +470,7 @@ alfresco-deployment-service:
   image:
     repository: quay.io/alfresco/alfresco-deployment-service
     tag: develop
-    pullPolicy: Always
+    pullPolicy: IfNotPresent
```

Comment:
this line is fine, but i don't see the env var below.

Reply:
sorry this line above is for deployment-service image, let's remove it form this pr please
as we can't do that for deployment-service image but not for modeling-service and apps